nesta/core/schemas/tier_1/mappings/datasets/gtr_mapping.json

```diff
@@ -0,0 +1,115 @@
+{
+    "mappings": {
+        "dynamic": "strict",
+        "properties": {
+            "date_end_project": {
+                "type": "date"
+            },
+            "date_start_project": {
+                "type": "date"
+            },
+            "json_funding_project": {
+                "properties": {
+                    "amount": {
+                        "type": "integer"
+                    },
+                    "category": {
+                        "type": "keyword"
+                    },
+                    "currencyCode": {
+                        "type": "keyword"
+                    },
+                    "end": {
+                        "type": "date"
+                    },
+                    "start": {
+                        "type": "date"
+                    }
+                },
+                "type": "nested"
+            },
+            "json_outcomes_project": {
+                "dynamic": true,
+                "type": "nested"
+            },
+            "name_leadOrgDepartment_project": {
+                "fields": {
+                    "keyword": {
+                        "type": "keyword"
+                    }
+                },
+                "type": "text"
+            },
+            "name_of_funder": {
+                "fields": {
+                    "keyword": {
+                        "type": "keyword"
+                    }
+                },
+                "type": "text"
+            },
+            "status_of_project": {
+                "type": "keyword"
+            },
+            "terms_continent_project": {
+                "type": "keyword"
+            },
+            "terms_countries_project": {
+                "type": "keyword"
+            },
+            "terms_instituteIds_project": {
+                "type": "keyword"
+            },
+            "terms_institutes_project": {
+                "type": "keyword"
+            },
```

Comment:
Not sure about this one, should we be able to search it? If so I'd use text+keyword